TAG-S&P-questionnaire.md

```diff
@@ -0,0 +1,106 @@
+# Self-Review Questionnaire: Security and Privacy
+
+Questions from w3.org/TR/security-privacy-questionnaire
+
+## 2. Questions to Consider
+
+### 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+If unpartitioned third-party cookies are still in use, this feature does not expose any additional information that is not already available if sites were already using an unpartitioned cross-site cookie.
+If unpartitioned third-party cookies are blocked, this provides sites in third-party contexts the ability to use HTTP cookies that are scoped to a single top-level site.
+
+### 2.2. Is this specification exposing the minimum amount of information necessary to power the feature?
+
+Yes. This proposal gives the ability for third-party contexts the ability to use a familiar HTTP state primitive for use cases restricted to a users' activity in only the current top-level site.
+
+### 2.3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
+
+This proposal requires that cookies which use the Partitioned attribute also have the __Host- prefix.
```

Comment:
Start by pointing out that a server might put PII into a cookie, or attach a unique-ID-in-a-cookie to PII on the server, so it's important to protect that from network eavesdroppers. 

This is probably also a good place to brag that partitioned cookies can't share that PII between different top-level sites.

Reply:
> Start by pointing out that a server might put PII into a cookie, or attach a unique-ID-in-a-cookie to PII on the server, so it's important to protect that from network eavesdroppers.

Done.

> This is probably also a good place to brag that partitioned cookies can't share that PII between different top-level sites.

Done.